arsceneview/src/main/java/io/github/sceneview/ar/interaction/ArNodeManipulator.kt

```diff
@@ -58,8 +61,13 @@ open class ArNodeManipulator(
         selectedNode?.takeIf {
             currentGestureTransform == EditableTransform.POSITION && it.positionEditable
         }?.let { node ->
-            node.anchor()
+            lastHitResult?.takeIf { it.isTracking }?.let {
```

Comment:
I think it is a very rear case, however, the node can remain not anchored if the trackable isn't tracking.